recipe/meta.yaml

```diff
@@ -21,11 +21,13 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.8
```

Comment:
This avoids relying on `importlib-metadata`, which usually is a source of resolution conflicts (see https://github.com/PyCQA/pydocstyle/blob/master/pyproject.toml#L25)

Reply:
Well, I'm not sure how including a dependency actually adds `pip check` problems, but since conda-forge is [dropping 3.7 support](https://conda-forge.org/docs/user/announcements.html#dropping-python-3-7), I'm good with this.

Reply:
So the problem is different packages have different constraints on importlib-metadata and conda/mamba end up solving a set of dependencies that sometimes is not the one really required by the package you want to install. And then, `pip check` fails.